### PR TITLE
fixes #8956, #9337 - remove unused scripts and files from specfile

### PR DESCRIPTION
**Pull request by mccun934**

### CODE REVIEW

File: deploy/katello.spec [hunk 1]
```diff
@@ -126,21 +125,9 @@ Requires:       %{?scl_prefix}rubygem-foreman_gutterball
...
-TOKEN=/etc/katello/secret_token
-# this file must not be world readable at generation time
-umask 0077
-test -f $TOKEN || (echo $(</dev/urandom tr -dc A-Za-z0-9 | head -c128) > $TOKEN \
-    && chmod 600 $TOKEN && chown katello:katello $TOKEN)
-
-usermod -a -G katello-shared tomcat
-
 %files common
 %attr(600, katello, katello)
```
Comment (ehelms) [inline deploy/katello.spec]: Can we drop this too since we have eliminated katello user?

Comment (mccun934) [inline deploy/katello.spec]: gone ...

**Comment by ehelms** [comment]
Looks like this just needs a rebase.

**Comment by mccun934** [comment]
@ehelms rebased ..

**Comment by ehelms** [comment]
APJ

**Comment by ehelms** [comment]
[test]

**Pull request merged**